src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -100,6 +101,7 @@ protected override void Apply(TransactionOperationContext context, BlittableJson
                         break;
 
                     case nameof(DeleteValueCommand):
+                    case nameof(DeactivateLicenseCommand):
```

Comment:
Shouldn't we also have some code to handle the behavior of license disactivation?